cloud/providers/ec2/ec2.go

```diff
@@ -93,7 +94,7 @@ func (cloudManager *EC2Manager) IsSSHReachable(host *host.Host, keyPath string)
 	if err != nil {
 		return false, err
 	}
-	return hostutil.CheckSSHResponse(host, sshOpts)
+	return hostutil.CheckSSHResponse(context.TODO(), host, sshOpts)
```

Comment:
Should this be `context.Background`, since we're actually using it for something--deriving another context from it--in `CheckSSHResponse`?

Reply:
I think long term, we should pass a context into the IsSSHReachable method, but I didn't want to change the interface for this here, and I think TODO is the right way to tell ourselves that we're not done plumbing contexts through